zerver/views/__init__.py

```diff
@@ -1128,9 +1131,25 @@ def update_realm(request, user_profile, name=REQ(validator=check_string, default
                  restricted_to_domain=REQ(validator=check_bool, default=None),
                  invite_required=REQ(validator=check_bool, default=None),
                  invite_by_admins_only=REQ(validator=check_bool, default=None),
-                 create_stream_by_admins_only=REQ(validator=check_bool, default=None)):
-    # type: (HttpRequest, UserProfile, Optional[str], Optional[bool], Optional[bool], Optional[bool], Optional[bool]) -> HttpResponse
+                 create_stream_by_admins_only=REQ(validator=check_bool, default=None),
+                 allow_message_editing=REQ(validator=check_bool, default=None),
+                 message_edit_duration_seconds=REQ(validator=check_string, default=None)):
```

Comment:
I think you probably want to use `REQ(converter=to_non_negative_int)` (with that, you won't need the assert below).